docs/user/index.rst

```diff
@@ -0,0 +1,45 @@
+User Guide
+==========
+
+This section of the documentation provides information about how to install
+uvloop as well as how to use uvloop.
+
+
+Installation
+------------
+
+`uvloop` is available from PyPI. It requires Python 3.5 and Cython.
+
+Use pip to install it.
+
+.. code-block:: console
+
+    $ pip install uvloop
+
+.. note::
+
+    Use the latest version of Cython to minimise the potential for build
+    issues.
```

Comment:
Again, this is only relevant for dev builds. And I think I'll add a `requirements.txt` file, and maybe add a `make` target to setup the dev venv.